junit-platform-testkit/src/main/java/org/junit/platform/testkit/engine/Events.java

```diff
@@ -336,4 +398,60 @@ private static void assertEventsMatchExactly(List<Event> events, Condition<? sup
 		softly.assertAll();
 	}
 
+	@SafeVarargs
+	private static void assertEventsMatchLoosely(List<Event> events, Condition<? super Event>... conditions) {
+		SoftAssertions softly = new SoftAssertions();
+		for (Condition<? super Event> condition : conditions)
```

Comment:
Please add curly brackets for the for-loop.

Reply:
done